run

```diff
@@ -806,10 +806,9 @@ test() {
 
 install_python_test_deps_() {
     if [ -n "${CONDA_PREFIX}" ]; then
-        # https://github.com/rapidsai/cucim/pull/349#issuecomment-1203335731
-        # Do not update or change already-installed dependencies.
-        run_command conda install -c conda-forge -y --freeze-installed \
-            --file ${TOP}/python/cucim/requirements-test.txt
+        # Install test dependencies from pip because conda does not have all the dependencies (e.g., opencv-python).
```

Comment:
conda does have it, but the package is not called `opencv-python`. It was originally just `opencv` on `conda-forge` and included the python bindings, but this has apparently changed recently to `libopencv` with a `py-opencv` meta-package as described here:
https://github.com/conda-forge/opencv-feedstock/blob/4129446ade9cb1928fd40897960ed32a21c7f771/recipe/meta.yaml#L4-L21

That said, I am fine leaving this as a pip install here

Reply:
I see you later changed to `opencv-python-headless` for which I don't think there is a conda equivalent